apm-agent-plugins/apm-error-logging-plugin/src/main/java/co/elastic/apm/agent/error/logging/Log4j2LoggerErrorCapturingInstrumentation.java

```diff
@@ -29,18 +29,19 @@
 
 import java.util.Collection;
 
+import static co.elastic.apm.agent.error.logging.Slf4jLoggerErrorCapturingInstrumentation.SLF4J_LOGGER;
 import static net.bytebuddy.matcher.ElementMatchers.hasSuperType;
 import static net.bytebuddy.matcher.ElementMatchers.named;
 import static net.bytebuddy.matcher.ElementMatchers.not;
 
-public class Log4jLoggingInstrumentation extends AbstractLoggingInstrumentation {
+public class Log4j2LoggerErrorCapturingInstrumentation extends AbstractLoggerErrorCapturingInstrumentation {
 
-    // prevents the shade plugin from relocating org.slf4j.Logger to co.elastic.apm.agent.shaded.slf4j.Logger
-    private static final String SLF4J_LOGGER = "org!slf4j!Logger".replace('!', '.');
+    // prevents the shade plugin from relocating org.apache.logging.log4j.Logger to co.elastic.apm.agent.shaded.logging.log4j.Logger
+    static final String LOG4J2_LOGGER = "org!apache!logging!log4j!Logger".replace('!', '.');
```

Comment:
🤦  How come the test passed? We shade the tests as well?

Reply:
We don’t shade the test, that’s why it succeeded in tests.

Reply:
Ohh, I guess you mean we test with non shaded agent code...